t/plugin/prometheus.t

```diff
@@ -809,3 +809,90 @@ qr/apisix_/
 qr/etcd/
 --- no_error_log
 [error]
+
+
+
+=== TEST 45: create service with name
```

Comment:
1. New test should be added to prometheus2.t 
2. Test 45 & Test 46 should be merged into one
3. We need another two tests: one for route with name & one for name preferred but name missed.

Reply:
Fixed. PTAL